src/components/AttachmentPicker/propTypes.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /**
+     * A renderProp with the following interface
+     *
+     * @example
+     * <AttachmentPicker>
+     * {({openPicker}) => (
+     *     <Button
+     *         onPress={() => {
+     *             openPicker({
+     *                 onPicked: (file) => {
+     *                     // Display or upload File
+     *                 },
+     *             });
+     *         }}
+     *     />
+     * )}
+     * </AttachmentPicker>
+     * */
+    children: PropTypes.func.isRequired,
+};
```

Comment:
This separate file seems unnecessary for a single PropType

Reply:
I've done this because:
1. the component has platform specific implementations that need to keep the same (public) props interface. 
2. this was copy pasted to both implementations - if more props are added they will have to be copy pasted as well 
3. the example usage was copy pasted too - this means that if it changes for some reason it's possible to forget to update it on both places: e.g. ["This is the web/mWeb/desktop version since..."]( https://github.com/Expensify/Expensify.cash/pull/2114/files#diff-c5016cd4c4354cbb41b3252743d1d0024af50be56ecf42566893ad7052492b2bL104) written on the native component

Reply:
@Julesssss can we resolve this one?

 My goal was to save some repetition for the documentation

The props possible to pass to a component should always match the same interface no matter the underlying platform implementation. So for platform specific components you'd need to keep them in sync if they are defined in separate files

Anyway I can define two separate prop types if you say so

Reply:
Ah okay, I missed that it was reused -- I suppose that makes sense.